### PR TITLE
Bookyquickdirty

### PR DESCRIPTION
**Pull request by yangdikun**
Finished bookyquickdirty. Wrap up this branch.

### CODE REVIEW

**Comment by lheagy** [comment]
Hi Dikun, a couple things to clean up before reviewing this content, could you 
- [ ] merge master so that it is up-to-date
- [ ] update title underlines so that the testing passes (https://travis-ci.org/ubcgif/em/builds/114345526) 

Thanks!
build 929 - ubcgif em - travis ci 2016-03-08 17-33-42

File: content/case_histories/bookpurnong/setup.rst [hunk 1]
```diff
@@ -2,3 +2,39 @@
 
 Setup
 =====
+
+The survey area under investigation is along Murray River in the Bookpurnong Irrigation District of the Riverland region of South Australia (Figure :numref:`booky-location`). The irrigation on the river bank (high land) has recharged the groundwater and formed a hydraulic gradient towards the floodplain (Figure :numref:`booky-hydro`). The groundwater flow from the high land to the the floodplain brings in substantial amount of natural saline water, causing tree health problems (Figure :numref:`deadtree`). In order to preserve the ecosystem, Bookpurnong Salt Interception Scheme (SIS) is implemented to experiment the water management methods that use pumping and flooding, with a hope of replacing the saline water with fresh water. 
+
+
```
Comment (thast) [inline content/case_histories/bookpurnong/setup.rst]: Figure are not referenced, numbered. I've found them too small too (need to click and expand to see where the area actually is)

File: content/case_histories/bookpurnong/setup.rst [hunk 2]
```diff
@@ -2,3 +2,39 @@
...
+.. figure:: images/booky-deadtree.jpg
+    :align: center
+    :scale: 30% 
+    :name: deadtree
+
+
+
+
+In hydrology, the interaction between the river water and ground water can be in two modes. If the groundwater table is lower, the river water recharges the groundwater aquifer, cansuing the river to "lose" water. If the groundwater table is higher, the groundwater recharges the river, casuing the river to "gain" water (Figure :numref:`lose-gain`). At Bookpurnong, the Murray River floodplain salinization is asscoiated with "gaining" saline water from the irrigated highland. So the geological question here is where Murray River loses or gains water, and whether the SIS improves the salinization. These questions can be partially answered by hydrological drilling and sampling, but those data are spacitally confined. Fortunately, geophysical remote sensing is able to provide an overall understanding of salinization over the entire area, because their is direct correlation between the salinity and electrical conductivity (see next page on Property).
```
Comment (thast) [inline content/case_histories/bookpurnong/setup.rst]: recharges the groundwater aquifer, **causing** the river to "lose" water

Comment (thast) [inline content/case_histories/bookpurnong/setup.rst]: "see next page on Property" : add a link

File: content/case_histories/bookpurnong/setup.rst [hunk 3]
```diff
@@ -2,3 +2,39 @@
...
+    :scale: 30% 
+    :name: deadtree
+
+
+
+
+In hydrology, the interaction between the river water and ground water can be in two modes. If the groundwater table is lower, the river water recharges the groundwater aquifer, cansuing the river to "lose" water. If the groundwater table is higher, the groundwater recharges the river, casuing the river to "gain" water (Figure :numref:`lose-gain`). At Bookpurnong, the Murray River floodplain salinization is asscoiated with "gaining" saline water from the irrigated highland. So the geological question here is where Murray River loses or gains water, and whether the SIS improves the salinization. These questions can be partially answered by hydrological drilling and sampling, but those data are spacitally confined. Fortunately, geophysical remote sensing is able to provide an overall understanding of salinization over the entire area, because their is direct correlation between the salinity and electrical conductivity (see next page on Property). 
+
+
+.. figure:: images/booky-losegain.jpg
```
Comment (thast) [inline content/case_histories/bookpurnong/setup.rst]: it's a nice figure. a bit bigger would help to analyze it

File: content/case_histories/bookpurnong/properties.rst [hunk 4]
```diff
@@ -3,3 +3,25 @@
 Properties
 ==========
 
+The sandy topsoil at Bookpurnong is mostly saturated by water from the river, irrigation, and other source of recharge. The ground conductivity is therefore dominated by the salinity of groundwater. High salinity in water means more ions acting as free charge carriers, so high electrical conductivity is expected at the place suffers from high salinization (see more information under Physical Properties > Electrical Conductivity). At Bookpurnong, the fresh water is from the river and irrigation, and the saline water is from the natural acquifer under the highland. Therefore, the conductivity can be a direct indicator of the source of groundwater, from the river or the intrusion.
```
Comment (thast) [inline content/case_histories/bookpurnong/properties.rst]: " high electrical conductivity is expected at the place **suffering** from high salinization"

Comment (thast) [inline content/case_histories/bookpurnong/properties.rst]: "see more information under Physical Properties > Electrical Conductivity" : add a link

File: content/case_histories/bookpurnong/properties.rst [hunk 5]
```diff
@@ -3,3 +3,25 @@
...
+
+For the purpose of studying the effect of SIS, hydrogeologists have sampled and measured the electrical conductivity of the groundwater along several transects across the river floodplain (Figure :numref:`booky-drills`). The measured conductivity from several wells along Transect 3 is plotted as cross sections from the river bed to the bank both before and after SIS (Figure :numref:`booky-salinity`). 
+
+
+.. figure:: ./images/booky-drills.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-drills
+
```
Comment (thast) [inline content/case_histories/bookpurnong/properties.rst]: figures does not seemed numbered.

File: content/case_histories/bookpurnong/properties.rst [hunk 6]
```diff
@@ -3,3 +3,25 @@
...
+.. figure:: ./images/booky-salinity.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-salinity
```
Comment (thast) [inline content/case_histories/bookpurnong/properties.rst]: On my laptop this figure is larger than the screen. Do you have the same effect?

File: content/case_histories/bookpurnong/survey.rst [hunk 7]
```diff
@@ -3,3 +3,67 @@
 Survey
 ======
 
+Loop-loop system
+----------------
+
+DC resistivity can measure the ground conductivity (see Case History > Mt. Isa), but it requires direct connection between the instrument and the earth through cables and electrodes. Here we consider another type of measurement that relies on the inductive coupling between the instrument and the ground, a mechanism more suitable for portable or towed systems. Because it uses a loop as the transmitter and another loop as the receiver, it is often referred to as "loop-loop" system.
```
Comment (thast) [inline content/case_histories/bookpurnong/survey.rst]: (see Case History > Mt. Isa): Add a link

File: content/case_histories/bookpurnong/survey.rst [hunk 8]
```diff
@@ -3,3 +3,67 @@
...
+Loop-loop system
+----------------
+
+DC resistivity can measure the ground conductivity (see Case History > Mt. Isa), but it requires direct connection between the instrument and the earth through cables and electrodes. Here we consider another type of measurement that relies on the inductive coupling between the instrument and the ground, a mechanism more suitable for portable or towed systems. Because it uses a loop as the transmitter and another loop as the receiver, it is often referred to as "loop-loop" system. 
+
+In a loop-loop survey, the transmitter loop carries a time-varying current, which generates a time-varying primary magnetic field everywhere in a whole space. According to Faraday's law and Lenz's law, this changing magnetic field can induce a time-varying secondary current in the ground to resist the change of magnetic field. Then this secondary current can generate a time-varying secondary magnetic field that induced an electromotive force in the receiver loop. The capability of generating the induced current is determined by the ground conductivity. Generally speaking, more conductive ground tends to be more responsive. 
+
+Airborne EM
+-----------
+
```
Comment (thast) [inline content/case_histories/bookpurnong/survey.rst]: add a reference to a figure

File: content/case_histories/bookpurnong/survey.rst [hunk 9]
```diff
@@ -3,3 +3,67 @@
...
+* Skin depth
+* Footprint of a system
+* Separation of source and receiver
+* Resistive and conductive limits
+* Scaling with induction number
+* Height above the surface
+* Loop coupling
+* Data representation
+
+.. figure:: ./images/booky-resolve.jpg
```
Comment (thast) [inline content/case_histories/bookpurnong/survey.rst]: figure is too big on my laptop.
no figure number

File: content/case_histories/bookpurnong/survey.rst [hunk 10]
```diff
@@ -3,3 +3,67 @@
...
+* Frequency to time transform: DFT, digital filtering
+* Transmitter dipole moment
+* Data normalization
+
+.. figure:: ./images/booky-skytem.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-skytem
+
+The figure above shows the time-domain loop-loop system used at Bookpurnong. The commercial name of the system is SkyTEM, operated by SkyTEM Surveys ApS, Danmark. Its transmitter is a large loop and its two receiver loops are mounted at the rear of the frame, in orthogonal orientations measuring dB/dt in z (vertical) and x (in-line horizontal) directions. The final data are normalized as if the dipole moments of the transmitter and the receiver are both unity. One special feature of SkyTEM is its capability of operating in dual-mode - transmitting a high moment pulse and a low moment pulse sequentially. A high moment pulse has greater penetrating depth, but is difficult to cut off clearly in electronics, causing unaccepted bias in the early times. A low moment does not see very deep, but has cleaner early time channels. The high moment is 113000 NIA at 25 Hz and the low moment is 12560 NIA at 222.22 Hz at Bookpurnong.
```
Comment (thast) [inline content/case_histories/bookpurnong/survey.rst]: replace "figure above" with a reference to the figure

File: content/case_histories/bookpurnong/survey.rst [hunk 11]
```diff
@@ -3,3 +3,67 @@
...
+* Height above the surface
+* Loop coupling
+* Data representation
+
+.. figure:: ./images/booky-resolve.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-resolve
+    
+The figure above shows the frequency-domain loop-loop system used at Bookpurnong. The commercial name of the system is RESOLVE, operated by Fugro Airborne (now CGG). It operates five pairs of horizontal co-planar (HCP) loop-loop configurations at 382, 1822, 7970, 35920 and 130100 Hz, and one pair of vertical co-axial loop-loop configuration at 3258 Hz. The measured secondary magnetic field (Hs) is represented as the ratio to the primary field (Hp) in part per million.
```
Comment (thast) [inline content/case_histories/bookpurnong/survey.rst]: replace "figure above" with a reference to the figure

File: content/case_histories/bookpurnong/survey.rst [hunk 12]
```diff
@@ -3,3 +3,67 @@
...
+
+Time domain system
+------------------
+
+If the transmitter current waveform is not sinusoid, it is said to operate in time domain. Usually the time-domain transmitter waveform is designed as an abrupt pulse or step off (on-time) followed by a no-current quiet peroid (off-time). The turn-off of the primary field excites the secondary current and field in the ground. The receiver loop measures the time-decaying transient at a number of time channels during the off-time when the primary field is absent. In theory, a time-domain system is equivelant to the convolution of the transmitter waveform with the inverse Fourier transform of a multi-frequency frequency-domain system. Early time channels are roughly equal to higher frequencies. Unlike a frequency-domain system, the time-domain data re real values of the actual field. Direct measurement of the magnetic field (B in Tesla) is possible, but people usually measure the time derivatives of the magnetic field (dB/dt in volt) for logistic reasons. You may find the following concepts useful. Click the links to learn more in Geophysical Surveys.
```
Comment (thast) [inline content/case_histories/bookpurnong/survey.rst]: "the time-domain **data** read real values of the actual field."

Comment (thast) [inline content/case_histories/bookpurnong/survey.rst]: "In theory, a time-domain system is equivelant to the convolution of the transmitter waveform with the inverse Fourier transform of a multi-frequency frequency-domain system. " : the formulation seemed confusing for me.

File: content/case_histories/bookpurnong/survey.rst [hunk 13]
```diff
@@ -3,3 +3,67 @@
...
+
+
+Survey design
+-------------
+
+Choosing the instrument and the survey parameters and layout is usually referred to as "survey design". In most cases, a geophysical survey is designed using the rules of thumbs or emprical criteria. In an EM survey, the most important goal in the survey design is to match the scale of investigation to the scale of the target, using the concept of skin depth or diffusion distance. At Bookpurnong, the background conductivity is about 1 S/m, and we are interested in the subsurface from 0 to about 50 m depth. The skin depths of the RESOLVE frequencies range from about 1 to 25 m. The diffusion distances of the SkyTEM time channels range 4 to 120 m. So the interested volume of the Earth should be reasonablly sensed by the two systems.
```
Comment (thast) [inline content/case_histories/bookpurnong/survey.rst]: "designed using **[]** rules of thumbs or emprical criteria. "

File: content/case_histories/bookpurnong/survey.rst [hunk 14]
```diff
@@ -3,3 +3,67 @@
...
+
+
+Survey design
+-------------
+
+Choosing the instrument and the survey parameters and layout is usually referred to as "survey design". In most cases, a geophysical survey is designed using the rules of thumbs or emprical criteria. In an EM survey, the most important goal in the survey design is to match the scale of investigation to the scale of the target, using the concept of skin depth or diffusion distance. At Bookpurnong, the background conductivity is about 1 S/m, and we are interested in the subsurface from 0 to about 50 m depth. The skin depths of the RESOLVE frequencies range from about 1 to 25 m. The diffusion distances of the SkyTEM time channels range 4 to 120 m. So the interested volume of the Earth should be reasonablly sensed by the two systems. 
+
+Another consideration is the sounding spacing. For a towed system, the in-line spacing is adjustable by the flight speed and sampling rates, and can easily achieve high density soundings. The cross-line spacing is usually a trade-oof between the cost and resolution. At Bookpurnong, RESOLVE and SkyTEM fly in a 100 m cross-line spacing (Figure), so any single location can be at least sensed by two SkyTEM lines.
```
Comment (thast) [inline content/case_histories/bookpurnong/survey.rst]: ross-line spacing (Figure) : add a ref to figure

File: content/case_histories/bookpurnong/data.rst [hunk 15]
```diff
@@ -2,3 +2,82 @@
...
+
+Here we make distinction bewteen "raw data" and "delivered data". Raw data are usually direct dump from the instruments, and may contain repeated measurements at the same location. For EM, the raw data are generally in the form of time series at a certain sampling rate. Then the raw data are processed by stacking, averaging, transforming, filtering, etc. The delivered data are usually in a reduced format of smaller size and higher signal noise ratio. In our framework, we consider "data" as "delivered data".
+
+An important task in this particular step is to visualize the data. Data visualization serves several purposes:
+
+(1) Understanding the underlying physics.
+
+(2) Data quality control.
+
+(3) Quanlitative interpretation.
```
Comment (thast) [inline content/case_histories/bookpurnong/data.rst]: +(3) **Quantitative** interpretation.

File: content/case_histories/bookpurnong/data.rst [hunk 16]
```diff
@@ -2,3 +2,82 @@
...
+
+(3) Sounding: plotting data at a particular location as a function of time/freq. Drill the data volume vertically. 
+
+The RESOLVE and SkyTEM data are presented in those three formats below.
+
+
+RESOLVE
+-------
+
+The figure below is a map view of the RESOLVE data in ppm. The real and imaginary parts of 382 and 7970 Hz are shown. We can identify the area with relatively high responses (warm color) that are potentially associated with the saline water and low responses (cold color) for the fresher water. The strips are from the variation of flight height.
```
Comment (thast) [inline content/case_histories/bookpurnong/data.rst]: The figure below: replace with a ref to the figure

File: content/case_histories/bookpurnong/processing.rst [hunk 17]
```diff
@@ -2,3 +2,59 @@
...
+
+From an interpreter's point of view, data processing is a step to examine and edit the data so it can be reliably interpreted in the next step. There are several major reasons why processing is necessary:
+
+(1) Quality control. Data without quality or uncertainty assessment mean nothing. So it is important to know the overall quality of a data set. A data set may be deemed not suitable for interpretation if the noise level is too high. For most data sets, preliminary QC is carried out during acquisition. So the delivered data can still show useful signals in decent quality. But we still have to identify the "bad data". 
+
+(2) Uncertainty analysis. Uncertainty is a quantitative way of assessing the data qiality. Data with greater noise may be assigned larger uncertainty. Most inversion programs need this information to decide how well the inversion wants to fit a particular datum.
```
Comment (thast) [inline content/case_histories/bookpurnong/processing.rst]: data **quality**

File: content/case_histories/bookpurnong/processing.rst [hunk 18]
```diff
@@ -2,3 +2,59 @@
...
+
+(4) Model parameterization. Any interpretation is based on models. By processing the data, we may choose more proper models. For example, negative transients in a central loop TEM survey indicate the existance of induced polarization. So we know at some places a real and time-independent conductivity model is not enough to explain the data. Another example is the variation of data in space may indicate the scale of EM induction, which helps the design of discretization for numerical modeling.
+
+
+Processing of RESOLVE data
+--------------------------
+
+The RESOLVE data, viewed along the profiles, is mostly smooth, so the high-frequency random noise is likely to be suppressed at the contractor's end. But we are still able to identify some outliers with suspecious negative values. We have therefore removed them.
+
+.. figure:: ./images/booky-resolveqc.jpg
```
Comment (thast) [inline content/case_histories/bookpurnong/processing.rst]: I love this figure.

File: content/case_histories/bookpurnong/interpretation.rst [hunk 19]
```diff
@@ -2,3 +2,77 @@
...
+
+(4) Plate modeling. Some geologic targets can be characterized as conductive thin plates that give rise complicated EM responses due to the geometry and mutual coupling (Link to 3-loop). Plate modeling method attempts to find the geometry and conductivity of a few conducting plate in a uniform hackground that is responsible for most of the anomalous data. It has the advantages of being able to handle the 3D coupling effect efficiently, but may have trouble dealing with too many plates in more complex situations. Link to plate modeling page.
+
+(5) 1D layered earth model inversion. This approach assume the earth's codnuctivity only varies as a function of depth. At each measurement location, the inversion find a layered model that explains the entire decay curve in time or the entire spectrum in frequency-domain. Many layered models at multiple locations then can be stitched together to form a pseudo-3D volume for visualization. Advanced techniques also consider the correlation between adjacent locations by imposing lateral constraints, etc. Link to 1D inversion page.
```
Comment (thast) [inline content/case_histories/bookpurnong/interpretation.rst]: "This approach assume the earth's **conductivity** "

File: content/case_histories/bookpurnong/interpretation.rst [hunk 20]
```diff
@@ -2,3 +2,77 @@
...
+
+The geology at Bookpurnong is mostly flat strata of floodplain. Therefore, an 1D inversion would be the most cost-effective solution for the data interpretation. In the following, we will go through the 1D inversion of the three data sets. 
+
+
+1D inversion of RESOLVE data
+----------------------------
+
+For RESOLVE data, the earth is discretized into 21 layers over the basement; the layers' thickness logarithmically increases from 0.5 to 43 m. Two different values, 1 and 0.1 S/m, are used as the conductivity of the uniform background model for two independent inversion trials. 
+
+.. figure:: ./images/booky-resolvefit.jpg
```
Comment (thast) [inline content/case_histories/bookpurnong/interpretation.rst]: I like the figure but need a legend. What are the red vs blue? continue vs dashed lines? (pred. vs obs ; In-Phase / quadrature? ) what represent the two axis and what are the units?

File: content/case_histories/bookpurnong/interpretation.rst [hunk 21]
```diff
@@ -2,3 +2,77 @@
...
+    :name: booky-resolvemodel2
+    
+The cross section view of the models show how the conductivity varies at depth and laterally. The basic structure of conductivity is a more conductive layer overlain by a resistive layer on the top. The depth to the conductive layer, potentially the saline water lenses, is consistently estimated regardless of the background conductivity value. However, the thickness of the saline water and the conductivity beneath it are not well delineated. This is because the relatively high frequency band of RESOLVE cannot penetrate the conductive layer, so the conductivity below a certain depth is mostly determined by the prescribed background model. 
+
+1D inversion of SkyTEM(HM) data
+-------------------------------
+
+The SkyTEM inversion follows the same procedures in RESOLVE inversion. The earth is discretized in the same way. Two background conductivity values, 0.1 and 1 S/m are used. 
+
+.. figure:: ./images/booky-skytemfit.jpg
```
Comment (thast) [inline content/case_histories/bookpurnong/interpretation.rst]: Nice figure. but again would need axis legends.

File: content/case_histories/bookpurnong/synthesis.rst [hunk 22]
```diff
@@ -3,3 +3,45 @@
 Synthesis
```
Comment (thast) [inline content/case_histories/bookpurnong/synthesis.rst]: Very Nice part!

File: content/case_histories/bookpurnong/synthesis.rst [hunk 23]
```diff
@@ -3,3 +3,45 @@
...
+
+Possibilities are:
+
+(1) Systematic errors in the data. For example, the flight height could be incorrect.
+
+(2) Naturally occuring variation of the conductivity or due to SIS.
+
+(3) The weaker conductivity in SkyTEM may be compensated by the thickness. 
+
+So further investigation may be needed to reconcile the models.
```
Comment (thast) [inline content/case_histories/bookpurnong/synthesis.rst]: Boreholes to give local reference and constrains? it seemed we had some in the region

**Comment by thast** [comment]
Nice job Dikun!
Overall I just picked some spelling. Some figures may deserve even more legend/description.

Thanks for your work on it!

**Comment by sgkang** [comment]
Nice work @yangdikun!! There are a number of good materials, but I think that most of them can be moved to either fundamentals or geophysical surveys. In addition, possibly the most important question is "how the AEM can help characterizing hydrological and environmental systems". Hence, we should focus on illustrating materials for this purpose. 

Some of my detailed suggestions are:
- [ ] Generating much simpler geology figure 
- [ ] Move any general information in Survey, Data, Processing, and Interpretation to Geophysical Surveys (AEM) 

There are a number of todos that you put it in while making this case study, I'll raise those items as issues, hence those small items can be distributed.

**Comment by lheagy** [comment]
Before merging, 
- [ ] reviewer comments need to be addressed
- [ ] master needs to be merged in so that the branch is up to date
- [ ] large paragraphs should have line breaks for more transparent version control (I am happy to help here!)

File: content/case_histories/bookpurnong/setup.rst (hunk 1 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/setup.rst]: Placeholders. To be reproduced. To be determined by Doug.

File: content/case_histories/bookpurnong/setup.rst (hunk 2 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/setup.rst]: I don't know if it's the proper format of referring the readers to other pages "see next page on ???". @lheagy  ?

File: content/case_histories/bookpurnong/setup.rst (hunk 3 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/setup.rst]: To be reproduced.

File: content/case_histories/bookpurnong/properties.rst (hunk 5 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/properties.rst]: To be reproduced.

File: content/case_histories/bookpurnong/properties.rst (hunk 6 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/properties.rst]: Mine looks fine. This seems a common problem...

File: content/case_histories/bookpurnong/survey.rst (hunk 8 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/survey.rst]: To be reproduced.

File: content/case_histories/bookpurnong/survey.rst (hunk 9 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/survey.rst]: Mine looks fine...

File: content/case_histories/bookpurnong/survey.rst (hunk 10 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/survey.rst]: To be reproduced.

File: content/case_histories/bookpurnong/survey.rst (hunk 11 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/survey.rst]: To be reproduced.

File: content/case_histories/bookpurnong/survey.rst (hunk 12 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/survey.rst]: Likely to be rewritten in Geophysical Surveys.

File: content/case_histories/bookpurnong/survey.rst (hunk 14 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/survey.rst]: To be reproduced.

File: content/case_histories/bookpurnong/data.rst (hunk 15 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/data.rst]: Fixed.

File: content/case_histories/bookpurnong/data.rst (hunk 16 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/data.rst]: TBR

File: content/case_histories/bookpurnong/synthesis.rst (hunk 23 above)
Comment (yangdikun) [inline content/case_histories/bookpurnong/synthesis.rst]: Not really.

Comment (yangdikun) [inline content/case_histories/bookpurnong/synthesis.rst]: We don't have.

File: content/case_histories/bookpurnong/survey.rst (hunk 7 above)
Comment (lheagy) [inline content/case_histories/bookpurnong/survey.rst]: here you can use `` (see the :ref:`Mt. Isa Case History<mt_isa_index>)` ``to make a link (ie ``:ref:`Name on thing you want linked<link_anchor>`)``

**Comment by lheagy** [comment]
- [ ] need to update / remake figures so that we have permission to show them

**Pull request merged**